pkg/controller/configmap/configmap_controller.go

```diff
@@ -76,7 +100,8 @@ func add(mgr manager.Manager, sharedInfo *sharedinfo.SharedInfo, r reconcile.Rec
 	}
```

Comment:
Add byNameSpaceFilter for NcpInstall CRD?

Reply:
So far, I don't find NcpInstall CRD type resources in other namespaces other than nsx-system-operator, however, it's a good defensive strategy to add byNameSpaceFilter for NcpInstall CRD as well, since it's supposed that we want to watch NcpInstall CRD only in nsx-system-operator as well.